<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>994</width>
    <height>599</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">color: black;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>141</width>
      <height>130</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="lb_connection_2">
       <property name="text">
        <string>Запрос телеметрии</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_gps">
       <property name="text">
        <string>GPS</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_imu">
       <property name="text">
        <string>IMU</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="cb_auto_gps">
       <property name="text">
        <string>Авто запрос GPS</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="cb_auto_imu">
       <property name="text">
        <string>Авто запрос IMU</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>141</width>
      <height>78</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="lb_connection">
       <property name="text">
        <string>Подключение клиента</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_server_start">
       <property name="text">
        <string>Подкл</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_server_stop">
       <property name="text">
        <string>Откл</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>100</y>
      <width>141</width>
      <height>111</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>Выбор режима</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_mnl_mode">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>25</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>Ручной</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_rmt_mode">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>25</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>Дистанционный</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_auto_mode">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>25</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>Автономный</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>240</y>
      <width>141</width>
      <height>80</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <widget class="QLabel" name="lb_mtr_ctrl">
       <property name="text">
        <string>Управление двигателем</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_motor_start">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>25</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>Старт</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_motor_stop">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>25</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>Стоп</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>220</y>
      <width>141</width>
      <height>49</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_5">
     <item>
      <widget class="QLabel" name="lb_set_power">
       <property name="text">
        <string>Задание мощности</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="le_pwr_mnl">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>25</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>10</y>
      <width>141</width>
      <height>81</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_6">
     <item>
      <widget class="QLabel" name="lb_info">
       <property name="text">
        <string>Инфо</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="lb_status">
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string>Статус</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lb_status_val">
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string>Не подключен</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="lb_rmode">
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string>Режим</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lb_rmode_val">
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string>Не выбрано</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>330</y>
      <width>141</width>
      <height>80</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_7">
     <item>
      <widget class="QLabel" name="lb_man_cmd">
       <property name="text">
        <string>Ввод команды вручную</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="le_mnl_cmd">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>25</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_mnl_cmd">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>25</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>Отправить</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>10</y>
      <width>461</width>
      <height>261</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_8">
     <item>
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>Терминала</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTextBrowser" name="terminal_window">
       <property name="font">
        <font>
         <family>Segoe UI</family>
         <pointsize>8</pointsize>
         <italic>false</italic>
         <bold>false</bold>
         <kerning>true</kerning>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_clean_textBrw">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>25</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>Очистить</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>10</y>
      <width>126</width>
      <height>140</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_9">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="lb_latitude">
         <property name="minimumSize">
          <size>
           <width>58</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>58</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>Latitide</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lb_latitude_val">
         <property name="text">
          <string>Latitide</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="lb_NS">
         <property name="minimumSize">
          <size>
           <width>58</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>58</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>NS</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lb_NS_val">
         <property name="text">
          <string>NS</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QLabel" name="lb_longitude">
         <property name="minimumSize">
          <size>
           <width>58</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>58</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>Longitude</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lb_longitude_val">
         <property name="text">
          <string>Longitude</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QLabel" name="lb_EW">
         <property name="minimumSize">
          <size>
           <width>58</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>58</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>EW</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lb_EW_val">
         <property name="text">
          <string>EW</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_7">
       <item>
        <widget class="QLabel" name="lb_altitude">
         <property name="minimumSize">
          <size>
           <width>58</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>58</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>Altitude</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lb_altitude_val">
         <property name="text">
          <string>Altitude</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_8">
       <item>
        <widget class="QLabel" name="lb_grndspeed">
         <property name="minimumSize">
          <size>
           <width>58</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>58</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>GrndSpeed</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lb_grndspeed_val">
         <property name="text">
          <string>GrndSpeed</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>160</y>
      <width>131</width>
      <height>236</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_10">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_9">
       <item>
        <widget class="QLabel" name="lb_axl_x">
         <property name="minimumSize">
          <size>
           <width>58</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>58</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>AXL_x</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lb_axl_x_val">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>AXL_x</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_10">
       <item>
        <widget class="QLabel" name="lb_axl_y">
         <property name="minimumSize">
          <size>
           <width>58</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>58</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>AXL_y</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lb_axl_y_val">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>AXL_y</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_11">
       <item>
        <widget class="QLabel" name="lb_axl_z">
         <property name="minimumSize">
          <size>
           <width>58</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>58</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>AXL_z</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lb_axl_z_val">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>AXL_z</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_12">
       <item>
        <widget class="QLabel" name="lb_mag_x">
         <property name="minimumSize">
          <size>
           <width>58</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>58</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>MAG_x</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lb_mag_x_val">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>MAG_x</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_13">
       <item>
        <widget class="QLabel" name="lb_mag_y">
         <property name="minimumSize">
          <size>
           <width>58</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>58</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>MAG_y</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lb_mag_y_val">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>MAG_y</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_14">
       <item>
        <widget class="QLabel" name="lb_mag_z">
         <property name="minimumSize">
          <size>
           <width>58</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>58</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>MAG_z</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lb_mag_z_val">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>MAG_z</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_15">
       <item>
        <widget class="QLabel" name="lb_gyro_x">
         <property name="minimumSize">
          <size>
           <width>58</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>58</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>GYRO_x</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lb_gyro_x_val">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>GYRO_x</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_16">
       <item>
        <widget class="QLabel" name="lb_gyro_y">
         <property name="minimumSize">
          <size>
           <width>58</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>58</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>GYRO_y</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lb_gyro_y_val">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>GYRO_y</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_17">
       <item>
        <widget class="QLabel" name="lb_gyro_z">
         <property name="minimumSize">
          <size>
           <width>58</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>58</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>GYRO_z</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lb_gyro_z_val">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>GYRO_z</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_18">
       <item>
        <widget class="QLabel" name="lb_grn_head">
         <property name="minimumSize">
          <size>
           <width>58</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>58</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>GndHead</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lb_grn_head_val">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>GndHead</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>994</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
